### PR TITLE
Feature/minor improvements in references

### PR DESCRIPTION
**Pull request by romaninsh**
Also fixes build problem.

### CODE REVIEW

File: src/Reference_SQL_One.php [hunk 1]
```diff
@@ -14,25 +14,33 @@ class Reference_SQL_One extends Reference_One
...
+     * @param string|null        $their_field
      *
      * @return Field_SQL_Expression
      */
-    public function addField($field, $their_field = null, $defaults = [])
+    public function addField($field, $their_field = null)
     {
+        if (is_array($field)) {
+            $defaults = $field;
+            $field = $defaults[0];
```
Comment (DarkSide666) [inline src/Reference_SQL_One.php]: probably should check if $defaults[0] is set at all

File: src/Reference_SQL_One.php [hunk 2]
```diff
@@ -43,20 +51,39 @@ function ($m) use ($their_field) {
...
      *
      * @return $this
      */
-    public function addFields($fields = [])
+    public function addFields($fields = [], $defaults = [])
     {
         foreach ($fields as $field => $alias) {
-            if (is_numeric($field)) {
-                $this->addField($alias);
+            $d = $defaults;
```
Comment (DarkSide666) [inline src/Reference_SQL_One.php]: not needed row

File: src/Reference_SQL_One.php [hunk 3]
```diff
@@ -43,20 +51,39 @@ function ($m) use ($their_field) {
...
+
+            if (is_array($alias)) {
+                $d = array_merge($defaults, $alias);
+                $alias = $alias[0];
```
Comment (DarkSide666) [inline src/Reference_SQL_One.php]: shouldn't we first check if $alias[0] is set at all?

File: src/Reference_SQL_One.php (hunk 1 above)
Comment (romaninsh) [inline src/Reference_SQL_One.php]: it's a bit meaningless not to have there, but ok.

File: src/Reference_SQL_One.php (hunk 2 above)
Comment (romaninsh) [inline src/Reference_SQL_One.php]: why not

Comment (romaninsh) [inline src/Reference_SQL_One.php]: adressed.

File: src/Reference_SQL_One.php (hunk 3 above)
Comment (romaninsh) [inline src/Reference_SQL_One.php]: fixed.

File: src/Reference_SQL_One.php (hunk 1 above)
Comment (romaninsh) [inline src/Reference_SQL_One.php]: ok, you are right, fixed.

**Pull request merged**